install.sh

```diff
@@ -20,41 +25,38 @@ declare -a hooks=(
     "pre-commit"
 )
 
+# providers to be installed
+declare -a providers=(
+    "commons"
+)
+
+# get script directory
+script_directory="$(dirname $0)"
+
 # install each hook
+mkdir -p ".git/hooks"
 for hook in "${hooks[@]}"
 do
     dest=".git/hooks/$hook"
-    if [ -z "$is_local" ]; then
-        url="https://raw.githubusercontent.com/$repo/$branch/hooks/$hook"
-        echo "Download from $url"
-        curl "$url" -o "$dest"
-    else 
-        script_directory="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
-        file="$script_directory/hooks/$hook"
-        echo "Copying $file"
-        cp "$file" "$dest"
-    fi
-
+    file="$script_directory/hooks/$hook"
+    echo "Copying hook file: $file"
+    cp "$file" "$dest"
     chmod +x "$dest"
 done
 
-# Install git-secrets command
-function command_exists {
-  #this should be a very portable way of checking if something is on the path
-  #usage: "if command_exists foo; then echo it exists; fi"
-  type "$1" &> /dev/null
-}
-
-if [ ! -f /usr/local/bin/git-secrets ]; then 
-    echo "You don't have git secrets installed. It will be used by some hooks."
-    read -p "Would you like to install git secrets (y/n)? " -n 1 -r
-    echo   
-    if [[ $REPLY =~ ^[Yy]$ ]]; then
-        echo "Installing git-secrets"
-        download_dir="/tmp/git-secrets-$(date +%s)"
-        git clone --depth 1 https://github.com/awslabs/git-secrets.git $download_dir
-        cd $download_dir || exit
-        make install
-        echo "git-secrets installed. Please see https://github.com/awslabs/git-secrets for command usage."
-    fi
-fi
```

Comment:
@pp-ps why remove this part? Any security concerns?

Reply:
I think this would be a forced installation method that I'm trying to avoid. In addition from building from source code, users may want to use brew (https://formulae.brew.sh/formula/git-secrets) or an already built package on their distro of choice, if available (Ubuntu Eoan 19.10 has it). Also checking in a hardcoded path seems Linux, if not distro specific.